lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @use '../../scss-shared/nx-header-variables';
+
+.nx-toast-container {
+  grid-area: content;
+  position: static;
+  justify-self:end;
+  margin-right: 8px;
+  width: 500px;
+  z-index: 1060;
```

Comment:
`z-index`es are defined in `_nx-variables.scss` in order to make it see how they related to each other.

Reply:
> `z-index`es are defined in `_nx-variables.scss` in order to make it see how they related to each other.

added in the z-index variable here: [3deee00](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/3deee004bfc1d2460397ade2edc819b2cbbc6412)